apps/vr-tests-react-components/screener.config.js

```diff
@@ -17,26 +17,31 @@ function getCurrentHash() {
 
   return '';
 }
+/**
+ *
+ * @param {string} screenerApiKey
+ * @param {string} sourceBranchName
+ * @param {string} deployUrl
+ * @param {string} targetBranch
+ * @returns
+ */
+function getConfig(screenerApiKey, sourceBranchName, deployUrl, targetBranch) {
```

Comment:
**chore:** can we change the api surface of this function? positional arguments are an antipattern as they prevent expansion. For example lets say I'd like to specify only 3rd argument, that would force me to call this like this `getConfig(undefined,undefined, 'my/branch' )` which is wrong.


the api should look like: `function getConfig(options){}`

Reply:
can surely be done. However, the arguments used now are always needed in order to correctly call the screener API and the refactoring using this factoring was done in order to limit the usage of environment variables in multiple files. Should the API still be changed in this situation?

Reply:
> However, the arguments used now are always needed in order to correctly call the screener API

as we can see from the implementation that's not entirely true, `targetBranch` seems to be optional.

> Should the API still be changed in this situation

yes please, as I said, we dont wanna introduce more antipatterns . ty